scripts/helm-docs.sh

```diff
@@ -8,4 +8,4 @@ echo "Running Helm-Docs"
 docker run \
     -v "$CHART_DIR:/helm-docs" \
     -u $(id -u) \
-    jnorwood/helm-docs:latest
+    jnorwood/helm-docs:v1.11.0
```

Comment:
This is because v1.11.0 is no longer the latest release, which causes some README files to not match what helm-doc autogenerates. See failing pipeline https://github.com/caraml-dev/helm-charts/actions/runs/6181966424/job/16780834025?pr=356